modules/govuk_ci/manifests/credentials.pp

```diff
@@ -61,9 +53,5 @@
   file {"${jenkins_home}/.pypirc":
     content => template('govuk_ci/pypirc.erb'),
   }
-
-  file { "${jenkins_home}/.npmrc":
-    content => template('govuk_ci/npmrc.erb'),
```

Comment:
to ensure that the file is removed set it to absent with a `FIXME`, and then remove the resource later on down the line. For eg:
```
file { 'foo':
  ensure => 'absent',
}
```